src/cloud-api-adaptor/test/e2e/assessment_helpers.go

```diff
@@ -370,25 +390,36 @@ func GetNodeNameFromPod(ctx context.Context, client klient.Client, customPod *v1
 	return getStringFromPod(ctx, client, customPod, getNodeName)
 }
 
+func GetPodsFromJob(ctx context.Context, t *testing.T, client klient.Client, job *batchv1.Job) (*v1.PodList, error) {
+	clientset, err := kubernetes.NewForConfig(client.RESTConfig())
+	if err != nil {
+		return nil, fmt.Errorf("GetPodFromJob: get Kubernetes clientSet failed: %v", err)
```

Comment:
Hi @stevenhorsman !

s/GetPodFromJob/GetPodsFromJob/

Reply:
This returns a PodList, so thats why I called it GetPods as it could return multiple

Reply:
sorry, I didn't look at the line properly!